packages/ckeditor5-essentials/ckeditor5-metadata.json

```diff
@@ -3,7 +3,7 @@
 		{
 			"name": "Essentials",
 			"className": "Essentials",
-			"description": "Includes all essential editing features like clipboard, <kbd>Enter</kbd> and <kbd>Shift</kbd>+<kbd>Enter</kbd>, typing and undo support.",
+			"description": "This package includes all essential editing features like the clipboard, <kbd>Enter</kbd> and <kbd>Shift</kbd>+<kbd>Enter</kbd> keystrokes, typing and the undo support.",
```

Comment:
Looks like the longer version is still here? Same below. Can you review?

Reply:
Addressed in commit.